src/main/webapp/resources/js/pages/password/PasswordReset.jsx

```diff
@@ -0,0 +1,197 @@
+      .catch((error) => {
+        setUpdateError(true);
+        setErrorMessages(error.data.errors);
+      });
+  };
+
+  return (
+    <Row justify="center">
+      <Col style={{ width: 600 }}>
+        <Row justify="center" style={{ marginBottom: SPACE_MD }}>
+          <img
+            src={setBaseUrl("/resources/img/irida_logo_light.svg")}
+            height={60}
+            alt={i18n("generic.irida.website")}
+          />
+        </Row>
+
+        {updateSuccess && (
+          <Alert
+            style={{ marginBottom: SPACE_SM }}
+            message={i18n("password.reset.success")}
+            className="t-reset-success-alert"
+            action={
+              <Button
+                type="link"
+                onClick={() => window.location.replace(setBaseUrl("/"))}
+              >
+                {i18n("password.reset.success.link")}
+              </Button>
+            }
+            type="success"
+            showIcon
+          />
+        )}
+
+        {updateError && (
+          <Alert
+            style={{ marginBottom: SPACE_SM }}
+            className="t-reset-error-alert"
+            message="The following errors occurred:"
+            description={
+              <Typography.Paragraph>
+                {
+                  <List>
+                    {Object.keys(errorMessages).map(function (keyName) {
+                      return (
+                        <List.Item key={keyName}>
+                          {errorMessages[keyName]}
+                        </List.Item>
+                      );
+                    })}
+                  </List>
+                }
+              </Typography.Paragraph>
+            }
+            type="error"
+            showIcon
+          />
+        )}
+
+        {!updateSuccess
+          ? passwordExpired && (
+              <Alert
+                style={{ marginBottom: SPACE_SM }}
+                className="t-password-expired-alert"
+                message={i18n("password.reset.password_expired")}
+                type="error"
+                showIcon
+              />
+            )
+          : null}
+
+        {!updateSuccess && (
+          <div>
+            <Alert
+              style={{ marginBottom: SPACE_SM }}
+              message={i18n("PasswordReset.alert.title")}
+              className="t-password-policy-alert"
+              description={
+                <Typography.Paragraph>
+                  <List
+                    header={i18n("PasswordReset.alert.description")}
+                    dataSource={passwordRules}
+                    renderItem={(item) => <List.Item>- {item}</List.Item>}
+                  />
+                </Typography.Paragraph>
+              }
+              type="info"
+              showIcon
+            />
+            <Form form={form} name="resetPasswordForm" size="large">
+              <Item
+                name="password"
+                rules={[
+                  {
+                    required: true,
```

Comment:
Why are you not doing full client side validation?

Reply:
Still need to add this in. Was going to talk to you about this

Reply:
Added in 777c017999d1d3f31fbaa33b11618ca753c55b3b